src/python/pants/option/options_fingerprinter_test.py

```diff
@@ -12,139 +14,138 @@
     file_option,
 )
 from pants.option.options_fingerprinter import OptionsFingerprinter
-from pants.testutil.test_base import TestBase
+from pants.testutil.rule_runner import RuleRunner
 from pants.util.contextutil import temporary_dir
 
 
-class OptionsFingerprinterTest(TestBase):
-    def setUp(self) -> None:
-        super().setUp()
-        self.options_fingerprinter = OptionsFingerprinter()
```

Comment:
Instead of `self.options_fingerprinter`, we now create a new `OptionsFingerprinter()` every time we need it. Turns out, all the methods are really classmethods, even though they're marked as instance methods.